src/ch04-02-references-and-borrowing.md

```diff
@@ -221,7 +221,7 @@ let mut x_ref = &x;
 #}
 ```
 
-Notice that `x_ref` has the @Perm{write} permission, while `*x_ref` does not. That means we can assign `x_ref` to a different reference (e.g. `x_ref = &y`), but we cannot mutate the pointed data (e.g. `*x_ref += 1`).
+Notice that `x_ref` has the @Perm{write} permission, while `*x_ref` does not. That means we can assign a different reference to the `x_ref` variable (e.g. `x_ref = &y`), but we cannot mutate the data it points to (e.g. `*x_ref += 1`).
 
 More generally, permissions are defined on **paths** and not just variables. A path is anything you can put on the left-hand side of an assignment. Paths include:
```

Comment:
Since this PR is clarifying the references and borrowing section, recommend revising the term path to be [place expressions](https://doc.rust-lang.org/stable/reference/expressions.html#place-expressions-and-value-expressions) throughout since borrowing doesn't seem to apply to all paths.  Doing so would address [issue 176](https://github.com/cognitive-engineering-lab/rust-book/issues/176).